Common/config/rule.hpp

```diff
@@ -0,0 +1,60 @@
+#include "taskbarappearance.hpp"
```

Comment:
Missing `#pragma once`. File didn't get added to the vcxproj either, so can't be found in the IDE.